Lesson_Materials/Lecture_09_Asynchronous_Execution/index.html

```diff
@@ -267,9 +267,9 @@
 });
 
 {
-  <mark>auto hostAcc = buf.get_host_access(cgh);</mark>
+  <mark>auto hostAcc = buf.get_host_access(sycl::write_only);</mark>
 
-  <mark>hostAcc[idx] = /* some computation */</mark>
+  <mark>hostAcc[index] = /* some computation */</mark>
```

Comment:
I guess changing the name here is to reduce confusion with the kernel from above: maybe we could change this to something like
```suggestion
  <mark>std::for_each(&hostAcc[0], &hostAcc[hostAcc.get_size()], [](auto e) { /* some computation */ });</mark>
```
e.g. showing verification of the data or something. That would involve a change to read_only in the previous section.

This isn't a needed change, but I did feel I had to go look up what `index` might be (and the answer is, it's not a thing).

Reply:
It did cross my mind that this could be confusing. We could also change it to
```suggestion
  <mark>hostAcc[/* some index */] = /* some computation */</mark>
```
or something.

Reply:
Either's fine, but it would be nice to make it a bit clearer.